lib/mirror_matchers.dart

```diff
@@ -60,14 +60,14 @@ class _HasProperty extends Matcher {
       item, Description mismatchDescription, Map matchState, bool verbose) {
     var reason = matchState == null ? null : matchState['reason'];
     if (reason != null) {
-      mismatchDescription.add(reason);
+      mismatchDescription.add(reason as String);
```

Comment:
which lint is this?

Reply:
I also set `implicit-casts: false` and fixed the easy ones.

There were too many weird cases to leave it on for the whole package, though – at least for now...